bin/console

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
```

Comment:
I know this is created automatically by bundler, but what do you think about adding this as a rake task?